src/test/components/MenuButtons.test.js

```diff
@@ -310,25 +329,30 @@ describe('app/MenuButtons', function() {
 });
 
 describe('<MetaInfoPanel>', function() {
-  function setup(profile: Profile, symbolicationStatus = 'DONE') {
+  function setup(profile: Profile) {
     jest.spyOn(Date.prototype, 'toLocaleString').mockImplementation(function() {
       // eslint-disable-next-line babel/no-invalid-this
       return 'toLocaleString ' + this.toUTCString();
     });
-    const resymbolicateProfile = jest.fn();
-
-    const renderResults = render(
-      <MetaInfoPanel
-        profile={profile}
-        resymbolicateProfile={resymbolicateProfile}
-        symbolicationStatus={symbolicationStatus}
-      />
-    );
 
-    return {
-      resymbolicateProfile,
-      ...renderResults,
-    };
+    const store = blankStore();
+
+    // Note that we dispatch this action directly instead of using viewProfile
+    // or loadProfile because we want to control tightly how symbolication is
+    // started in these tests.
+    store.dispatch({
+      type: 'PROFILE_LOADED',
+      profile,
+      pathInZipFile: null,
+      implementationFilter: null,
+      transformStacks: null,
+    });
```

Comment:
When using `viewProfile` or `loadProfile` the symbolication starts if the profile's `symbolicated` value is `false`. This isn't a big issue as we could wait for the end of the symbolication very easily, however the problem then is that `symbolicated` becomes `true`.

One test was especially testing what happens when `symbolicated` is `false` (`''handles the contradictory state of non-symbolicated profiles that are done'`) and I didn't want to remove it, so that's why I'm now dispatching this raw action directly.

Reply:
Suggestion: Hmm... this makes me a bit nervous that these tests will fail mysteriously in the future. Would it be possible to make the symbolication kick-off process more explicit? Perhaps it would be worth adding a `skipSymbolication: boolean` property to the `viewProfile` action. This would be test-only, but would make this behavior more explicit. Using `PROFILE_LOADED` sounds incidental, which could change in the future.

Reply:
yeah that could be a good solution, I'll look at it